### PR TITLE
US006 Adding Responsibilities Per Role - WEBUI

### PR DESCRIPTION
**Pull request by CharlesM988**
Adding in Responsibilities Per Role to view-job-spec.html

- Modified jobController.ts
- Modified jobService.ts
- Modified view-job-spec.html
- Created a responsibility model

This is pulling information in from a new API endpoint.

Screenshot 2023-12-13 at 14 23 43
Screenshot 2023-12-14 at 09 23 30
Screenshot 2023-12-14 at 09 23 41
Screenshot 2023-12-20 at 09 47 57

### CODE REVIEW

**Review by amchugh689 — APPROVED** [review]
tested and once again can confirm it does work as intended. :D

File: test/unit/RoleResponsTest.js
Comment (jkelso662) [inline test/unit/RoleResponsTest.js:1]: Is there a spelling mistake on this file?

File: test/unit/RoleResponsTest.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+            var error;
+
+            try {
+                await jobService.getRoleResponsibilityById(1705);
+            } catch (e) {
+                var error = e.message
+            }
+
+            expect(error).to.equal("Could not get role responsibilities by id");
```
Comment (jkelso662) [inline test/unit/RoleResponsTest.js]: Some mad double spacing going on here 😆 not a blocker - just take note - this is weird haha

**Comment by CharlesM988** [comment]
@jkelso662 Issues resolved :) !

**Review by jkelso662 — APPROVED** [review]
LGTM

**Pull request merged**